README.md

```diff
@@ -71,6 +71,10 @@ higher-level APIs:
    ```
    kind create cluster
    ```
+1. Download required binaries:
+  ```
+  ./scripts/download-binaries.sh
+  ```
```

Comment:
```suggestion
1. Download required binaries:
  This step is only necessary if you don't have `kubectl` and `helm` on your PATH!
  ```
  ./scripts/download-binaries.sh
  export PATH=$(pwd)/bin:${PATH}
  ```
```

Reply:
+1

Reply:
I'd like to have both `./scripts/download-e2e-binaries.sh` and `./scripts/download-binaries.sh` in the step.
The `download-e2e-binaries.sh` downloads `kind`.